### ISSUE TITLE
VPC creation should be optional

### ISSUE THREAD

**Issue by jmetzmeier** [pythian/opsviz#21]
We should add a param for a VPC id and only create the VPC if that param is not set.

**Comment by jonathandietz** [comment]
I agree. This should be a fairly high priority compared to other feature requests.
